pkg/agent/controller/traceflow/packetin.go

```diff
@@ -344,7 +348,17 @@ func (c *Controller) parsePacketIn(pktIn *ofctrl.PacketIn) (*crdv1beta1.Traceflo
 			}
 			ob.Action = crdv1beta1.ActionForwardedOutOfOverlay
 		} else if outputPort == gwPort { // noEncap
-			ob.Action = crdv1beta1.ActionForwarded
+			// TODO: merge this and above case if needed
```

Comment:
I feel like the TODO is not helpful without extra context. There is no information regarding why / when this merging would be needed.

Reply:
Changed to `TODO: merge this and above case if noEncap mode supports Egress feature`